modules/apps/announcements/announcements-web/src/main/java/com/liferay/announcements/web/internal/display/context/DefaultAnnouncementsDisplayContext.java

```diff
@@ -419,6 +433,45 @@ public boolean isTabs1Visible() {
 		return false;
 	}
 
+	private LinkedHashMap<Long, long[]>
+			_combinedSegmentsEntryRoleAndAnnouncementScopes()
+		throws PortalException {
+
+		LinkedHashMap<Long, long[]> announcementScopes =
```

Comment:
Why are you declaring the variable as LinkedHashMap instead of Map?

Reply:
Hi @adolfopa : I do not quite understand the question, but if it is just about the return type, so I can tell it's the return type of method 
`AnnouncementsUtil.getAnnouncementScopes()` also the require type of `scope` param of AnnouncementsEntryLocalServiceUtil.getEntriesCount() we call later on L284 and L289.